src/app/modules/filters/filters-content/filters-content.component.html

```diff
@@ -0,0 +1,21 @@
+<div>
+    <div class="filter">
+        <ccf-checkbox
+            label="Technologies"
```

Comment:
You need to also take into account when filter is updated outside of the component. Should look something like <ccf-checkbox [selection]="filter.technology"